testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -89,9 +107,15 @@ public static void StartWorkerNode()
         }
     }
 
-    // todo remove this once we have proper automation
-    public static bool IsUsingUTR()
+    public static void KillAllProcesses()
     {
-        return Environment.GetCommandLineArgs().Contains("-automated");
+        Debug.Log("Killing processes...");
+        foreach (var process in Processes)
+        {
+            Debug.Log($"Killing process {process.Id} with state {process.HasExited}");
+            process.Kill();
```

Comment:
from what I read here
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=net-5.0
kill will throw 
```
InvalidOperationException
The process has already exited.
```
if the process is already dead, I'm guessing we still want to handle that case?

Reply:
Good point, I'll handle this better.